tests/SerdeTestCases.php

```diff
@@ -1187,6 +1188,41 @@ public function datetime_fields_support_custom_output_format(): void
 
     }
 
+    #[Test]
+    public function unixtime_fields_support(): void
+    {
+        $s = new SerdeCommon(formatters: $this->formatters);
+
+        $timeString = '4 July 2022 14:22:22';
+        $zone = new \DateTimeZone('America/New_York');
+
+        $stamp = new \DateTime($timeString, $zone);
+        $stampImmutable = new \DateTimeImmutable($timeString, $zone);
+        $data = new UnixTimeExample(seconds: $stamp, milliseconds: $stampImmutable);
+
+        $serialized = $s->serialize($data, $this->format);
+
+        switch($serialized) {
```

Comment:
This is a bad approach for the test.  It will break as soon as we add another format.

If you look, most of the tests have a validate method they pair with, which is only implemented in child classes.  Specifically, the `ArrayBasedFormatterTestCase` class (I think that's what it's called).  Following that pattern would be much more reliable here, and more accurate.